src/iGenVar.cpp

```diff
@@ -3,8 +3,10 @@
 #include <seqan3/core/debug_stream.hpp>                     // for seqan3::debug_stream
 
 #include "modules/clustering/simple_clustering_method.hpp"  // for the simple clustering method
-#include "variant_detection/validator.hpp"                  // for class EnumValidator
 #include "structures/cluster.hpp"                           // for class Cluster
+#include "variant_detection/method_enums.hpp"               // for enum detection_, clustering_ and refinement_methods
```

Comment:
I think this include is not strictly needed because it's already included in `variant_detection.hpp` but it probably doesn't hurt and is more verbose this way.